lib/components/app/app-menu.js

```diff
@@ -39,8 +40,10 @@ class AppMenu extends Component {
     window.location.href = startOverUrl
   }
 
+  _toggleMailables = () => this.props.toggleMailables()
```

Comment:
Should this be a toggle action? The other items under this menu don't toggle.

Reply:
Since this method calls the action without any further arguments or logic, the toggleMailables prop can be used directly in the render method.